TealiumCrashModule/TealiumCrashModuleTests/test_tealium_crash/TealiumCrashModuleTests.swift

```diff
@@ -132,4 +132,9 @@ class MockDataLayer: DataLayerManagerProtocol {
     
     func deleteAll() { }
     
+    func add(data: [String : Any], expiry: Expiry) {
+    }
+    
+    func add(key: String, value: Any, expiry: Expiry) {
+    }
```

Comment:
are these function supposed to be empty?

Reply:
Yes, the whole class is empty as it's a mock class, and the methods are just added for the interface